shapely/io.py

```diff
@@ -138,10 +147,23 @@ def to_wkb(
     include_srid : bool, default False
         If True, the SRID is be included in WKB (this is an extension
         to the OGC WKB specification).
+    flavor : {"iso", "extended"}, default "extended"
+        Which flavor of WKB will be returned. The flavor determines how
+        extra dimensionality is encoded with the type number, and whether
+        SRID can be included in the WKB (ISO flavor does not support SRID
+        embedding). ISO flavor is "more standard" for 3D output.
+        The `from_wkb` function can read both flavors.
     **kwargs
         For other keyword-only arguments, see the
         `NumPy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
 
+    Notes
+    -----
+
+    Standard WKB is returned by setting ``output_dimension=2`` and
```

Comment:
The term "Standard" is potentially confusing here, as both ISO and EWKB are also standards.

Maybe rephrase to

```Python
Both flavors are equivalent when ``output_dimension=2`` and ``include_srid=False``.
```

Reply:
Yeah, I agree that "standard" not a great term. It is used by the GEOS docs, though (https://libgeos.org/specifications/wkb/), and so that was the reason for Mike to suggest it. 

But I like your phrasing, and that can maybe be moved into the flavor parameter explanation.